src/main/java/com/metamx/http/client/HttpClientInit.java

```diff
@@ -110,22 +110,70 @@ public static HttpClient createClient(ResourcePoolConfig config, final SSLContex
     );
   }
 
+  @Deprecated // use createClient directly
   public static ClientBootstrap createBootstrap(Lifecycle lifecycle, Timer timer)
   {
-    // Default from NioClientSocketChannelFactory.DEFAULT_BOSS_COUNT, which is private:
-    final int bossCount = 1;
+    final HttpClientConfig defaultConfig = HttpClientConfig.builder().build();
+    return createBootstrap(lifecycle, timer, defaultConfig.getBossPoolSize(), defaultConfig.getWorkerPoolSize());
+  }
+
+  @Deprecated // use createClient directly
+  public static ClientBootstrap createBootstrap(Lifecycle lifecycle)
+  {
+    final Timer timer = new HashedWheelTimer(new ThreadFactoryBuilder().setDaemon(true).build());
+    return createBootstrap(lifecycle, timer);
+  }
 
-    // Default from SelectorUtil.DEFAULT_IO_THREADS, which is private:
-    final int workerCount = Runtime.getRuntime().availableProcessors() * 2;
+  public static SSLContext sslContextWithTrustedKeyStore(final String keyStorePath, final String keyStorePassword)
```

Comment:
This happened because I was trying to keep the public methods together without a private one in the middle. I can swap them around though if you prefer. let me know.